### ISSUE TITLE
About Efficiency Evaluation

### ISSUE THREAD

**Issue by peicwang** [DeepBitsTechnology/DeepDi#9]
Hi,I have a question about efficiency evaluation. When evaluating the efficiency of deepdi, does it include the time for superset disassembly? In the figure in your paper, it can be seen that when the binary code section size is less than 500KB, the disassembly time of deepdi-gpu is less than 0.1s, which is quite excellent. I want to know if this time includes the superset disassembly part of the binary.  I am looking forward to your reply!

**Comment by Nifury** [comment]
Yes, the time spent in superset disassembly is included. That part was written in CUDA and can achieve high parallelism.
